cfme/networks/__init__.py

```diff
@@ -0,0 +1,20 @@
+class ValidateStatsMixin(object):
+    def validate_stats(self, expected_stats):
```

Comment:
This is nice, but should likely be abstracted to a bit higher level where its useful for things beyond this network module. We have validate_stats built-in to the base provider, maybe BaseEntity could include this mixin? A TODO would be nice for now.